graphql/schemas/Ecosystem/notifications.graphql

```diff
@@ -97,4 +97,13 @@ extend type Mutation {
         @field(
             resolver: "App\\GraphQL\\Ecosystem\\Mutations\\Notifications\\ReadAllNotification"
         )
+    sendNotificationBaseOnTemplate(
+        template_name: String!
```

Comment:
@FredPeal add notificationType

Reply:
We have multiples templates by notifications types